src/shared/components/CustomButton/styles.scss

```diff
@@ -0,0 +1,4 @@
+.customButtonImage {
+    width: 18px;
+    height: 18px;
+}
```

Comment:
This is to make sure that whatever size the image is, the "button" is basically the same size as the Copy+Download font awesome buttons.